src/components/layouts/collection-page-layout.tsx

```diff
@@ -478,30 +479,46 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
               {!isEmpty(podcast) && (
                 <CoursePodcast podcast={podcast} instructorName={full_name} />
               )}
-              {topics && (
-                <div className="mt-8 border border-gray-100 dark:border-gray-700 rounded-md p-5">
-                  <h2 className="text-lg font-semibold mb-3">
-                    What you'll learn
-                  </h2>
-                  <div className="prose dark:prose-dark">
-                    <ul className="grid md:grid-cols-2 grid-cols-1 md:gap-x-5">
-                      {topics?.map((topic: string) => (
-                        <li
-                          key={topic}
-                          className="text-gray-900 dark:text-gray-100 leading-6"
-                        >
-                          {topic}
-                        </li>
-                      ))}
-                    </ul>
+              <div
+                className={`grid md:gap-x-5 ${
+                  prerequisites && topics ? 'grid-cols-2' : 'grid-cols-1'
+                }`}
+              >
+                {topics && (
+                  <div className="mt-8 border border-gray-100 dark:border-gray-700 rounded-md p-5">
+                    <h2 className="text-lg font-semibold mb-3">
+                      What you'll learn
+                    </h2>
+                    <div className="prose dark:prose-dark">
+                      <ul className="grid grid-cols-1 md:gap-x-5">
+                        {topics?.map((topic: string) => (
+                          <li
+                            key={topic}
+                            className="text-gray-900 dark:text-gray-100 leading-6"
+                          >
+                            {topic}
+                          </li>
+                        ))}
+                      </ul>
+                    </div>
                   </div>
-                </div>
-              )}
+                )}
+                {prerequisites && (
+                  <div className="mt-8 border border-gray-100 dark:border-gray-700 rounded-md p-5">
+                    <h2 className="text-lg font-semibold mb-3">
+                      Prerequisites
+                    </h2>
+                    <div className="prose dark:prose-dark">
+                      <Prereqs prerequisites={prerequisites} />
```

Comment:
Made a Prereqs component because needed to handle whether a prereq has a `path` attribute on it or not